doc/rados/operations/health-checks.rst

```diff
@@ -28,162 +28,161 @@ Monitor
 DAEMON_OLD_VERSION
 __________________
 
-Warn if old version(s) of Ceph are running on any daemons.
-It will generate a health error if multiple versions are detected.
-This condition must exist for over mon_warn_older_version_delay (set to 1 week by default) in order for the
-health condition to be triggered.  This allows most upgrades to proceed
-without falsely seeing the warning.  If upgrade is paused for an extended
-time period, health mute can be used like this
-"ceph health mute DAEMON_OLD_VERSION --sticky".  In this case after
-upgrade has finished use "ceph health unmute DAEMON_OLD_VERSION".
+Warn if one or more old versions of Ceph are running on any daemons.
+A health check is raised if multiple versions are detected.
+This condition must exist for a period of time greater than ``mon_warn_older_version_delay`` (set to one week by default) in order for the
+health check to be raised. This allows most upgrades to proceed
+without the occurrence of a false warning. If the upgrade is paused for an extended
+time period, ``health mute`` can be used by running
+``ceph health mute DAEMON_OLD_VERSION --sticky``. Be sure, however, to run ``ceph health unmute DAEMON_OLD_VERSION`` after the upgrade has finished.
 
 MON_DOWN
 ________
 
-One or more monitor daemons is currently down.  The cluster requires a
-majority (more than 1/2) of the monitors in order to function.  When
-one or more monitors are down, clients may have a harder time forming
-their initial connection to the cluster as they may need to try more
+One or more monitor daemons are currently down. The cluster requires a
+majority (more than one-half) of the monitors in order to function. When
+one or more monitors are down, clients might have a harder time forming
+their initial connection to the cluster, as they might need to try more
 addresses before they reach an operating monitor.
 
-The down monitor daemon should generally be restarted as soon as
+As a general rule, the down monitor daemon should be restarted as soon as
 possible to reduce the risk of a subsequent monitor failure leading to
 a service outage.
 
 MON_CLOCK_SKEW
 ______________
 
 The clocks on the hosts running the ceph-mon monitor daemons are not
-sufficiently well synchronized.  This health alert is raised if the
+sufficiently well-synchronized. This health check is raised if the
 cluster detects a clock skew greater than ``mon_clock_drift_allowed``.
 
-This is best resolved by synchronizing the clocks using a tool like
+This issue is best resolved by synchronizing the clocks by using a tool like
 ``ntpd`` or ``chrony``.
 
 If it is impractical to keep the clocks closely synchronized, the
-``mon_clock_drift_allowed`` threshold can also be increased, but this
+``mon_clock_drift_allowed`` threshold can also be increased. However, this
```

Comment:
IMHO is is never impractical, since synchronization != accuracy.  I'm tempted to suggest removing any mention of relaxing the clock skew here, as no good can come of it.

Reply:
unfinished business